cli/cmd/inject.go

```diff
@@ -336,119 +337,119 @@ func (resourceTransformerInject) generateReport(reports []inject.Report, output
 	output.Write([]byte("\n"))
 }
 
-func (options *proxyConfigOptions) fetchConfigsOrDefault() (*cfg.All, error) {
+func (options *proxyConfigOptions) fetchConfigsOrDefault() (*linkerd2.Values, error) {
 	if options.ignoreCluster {
 		if !options.disableIdentity {
 			return nil, errors.New("--disable-identity must be set with --ignore-cluster")
 		}
 
-		install, err := newInstallOptionsWithDefaults()
-		if err != nil {
-			return nil, err
-		}
-
-		return install.configs(nil), nil
+		return linkerd2.NewValues(false)
 	}
 
-	api := checkPublicAPIClientOrExit()
-	config, err := api.Config(context.Background(), &public.Empty{})
+	checkPublicAPIClientOrExit()
```

Comment:
Do we actually need the public API client for anything?

Reply:
It is used to perform a subset of `healthchecks` to make sure the control plane is healthy and is usually in most command paths.